about.html

```diff
@@ -50,20 +50,17 @@
                 <p>Special thanks to <a href="https://github.com/FuzzyAvacado">John</a>, the people at hack night, and any contributors!</p>
             </section>
             <section>
-                <a name="upcoming"></a>
-                <h1>Upcoming features</h1>
-                <ul>
-                    <li>Google Calender Integration</li>
-                    <li>Ability to manually add a course</li>
-                    <li>Better phone support</li>
-                    <li>Prettier calender design + general design improvements</li>
-                    <li>Better code efficency, modularity, documentation, and elegancy</li>
-                </ul>
+                <a name="end"></a>
+                <h1>End of UCRSV</h1>
+                <p>We had plenty of neat things planned for UCRSV like a update on the design, ability to manually add courses, and the ability to add your schedule to your Google calendar. With the new switch to Banner however, things have become much more difficult. The schedule in Banner is much harder to copy and paste now, though more importantly Banner already visualizes your schedule for you. While Banner's calendar view leaves much to be desired, it still defeats the purpose of our website. UCRSV is a website for visualizing the old unreadable GROWL schedule. While it would be nice to have the planned features implemented, the foundation of what this website used to stand on has now been pulled from under its feet. <br><br>
+                We do realize that some people still use and prefer our format even to Banner, so we are looking to implement support for Banner's schedule.<br><br>
+                Making UCRSV has been lots of fun. Thanks to everyone who used it.
+                </p>
             </section>
             <section>
-                <a name="bugs"></a>
-                <h1>Found an bug? Want a feature?</h1>
-                <p>Submit an <a href="https://github.com/BradleyCai/ucr-schedule-visualizer/issues/new">issue</a> so we can fix it! <br><br>If you're a developer, feel free to <a href="https://github.com/BradleyCai/ucr-schedule-visualizer/">fork</a> us and submit a <a href="https://github.com/BradleyCai/ucr-schedule-visualizer/pulls">pull request</a> to contribute code to this project. Yay free software!</p>
+                <a name="dev"></a>
+                <h1>UCRSV Development</h1>
+                <p>Don't forget that UCRSV is open! If you'd like to pick up where we left off feel free to <a href="https://github.com/BradleyCai/ucr-schedule-visualizer/">fork</a> us and contact us on <a href="https://discord.gg/0vTWbxIL45ClG4mV">discord</a> if you have any questions. And who knows, maybe Banner might even use some of our code in the future.</p>
```

Comment:
Maybe change the last sentence to add "maybe Banner might even _add an API or_ even use some..."